examples/DIPDialect/correlation2D.cpp

```diff
@@ -126,7 +94,8 @@ bool testImplementation(int argc, char *argv[], std::ptrdiff_t x,
   Mat kernel1 = Mat(3, 3, CV_32FC1, laplacianKernelAlign);
 
   // Call the MLIR Corr2D function.
-  _mlir_ciface_corr_2d(input, kernel, output, x, y, 0);
+  dip::Corr2D(input, kernel, output, x, y,
+              dip::BOUNDARY_OPTION::REPLICATE_PADDING);
```

Comment:
Is there an example for `dip::BOUNDARY_OPTION::CONSTANT_PADDING` option? If not, can we add an example for it?